hacks/release_controller/accept.py

```diff
@@ -1,94 +1,131 @@
-    $ ./accept.py -r 4.4.0-rc.3
-                  -u 'https://prow.svc.ci.openshift.org/view/...origin-installer-e2e-gcp-upgrade/575'
-                  -m 'https://prow.svc.ci.openshift.org/view/...origin-installer-e2e-gcp-upgrade/461'
-                  --confirm
+    $ ./accept.py -r 4.9.46 --confirm
     """
-
-    if not upgrade_minor_url and not upgrade_url:
-        click.echo('One or both upgrade urls must be specified in order to accept the release')
-        exit(1)
-
-    arch_suffix = ''
-    if arch != 'amd64' and arch != 'x86_64':
-        arch_suffix = f'-{arch}'
-
-    with oc.api_server(api_url='https://api.ci.l2s4.p1.openshiftapps.com:6443'), \
-         oc.project(f'ocp{arch_suffix}'):
-
-        istag_qname = f'istag/release{arch_suffix}:{release}'
-        istag = oc.selector(istag_qname).object(ignore_not_found=True)
-        if not istag:
-            raise IOError(f'Could not find {istag_qname}')
-
-        ts = int(round(time.time() * 1000))
-        backup_filename = f'release{arch_suffix}_{release}.{ts}.json'
-        if confirm:
-            with open(backup_filename, mode='w+', encoding='utf-8') as backup:
-                print(f'Creating backup file: {backup_filename}')
-                backup.write(json.dumps(istag.model._primitive(), indent=4))
-
-        def make_release_accepted(obj):
-            for annotations in (obj.model.image.metadata.annotations, obj.model.metadata.annotations, obj.model.tag.annotations):
-                annotations.pop('release.openshift.io/message', None)
-                annotations.pop('release.openshift.io/reason', None)
-                annotations['release.openshift.io/phase'] = 'Accepted' if not reject else 'Rejected'
-
-                verify_str = annotations['release.openshift.io/verify']
-                verify = oc.Model(json.loads(verify_str))
-                verify.upgrade.state = 'Succeeded' if not reject else 'Failed'
-                if upgrade_url:
-                    verify.upgrade.url = upgrade_url
-                verify['upgrade-minor'].state = 'Succeeded' if not reject else 'Failed'
-                if upgrade_minor_url:
-                    verify['upgrade-minor'].url = upgrade_minor_url
-                annotations['release.openshift.io/verify'] = json.dumps(verify._primitive(), indent=None)
-
-            print(json.dumps(obj.model._primitive(), indent=4))
-            if confirm:
-                print('Attempting to apply this object.')
-                return True
-            else:
-                print(WARNING + '--confirm was not specified. Run again to apply these changes.' + ENDC)
-                exit(0)
-
-        make_release_accepted(istag)
-        istag.replace()
-        print('Success!')
-        print(f'Backup written to: {backup_filename}')
+    if reject and not why:
+        raise click.BadParameter('--why (a reason) is required when rejecting a release')
+
+    kubeconfig = os.getenv('KUBECONFIG')
+    if confirm and not kubeconfig:
```

Comment:
I don't think this env var check is needed. `KUBECONFIG` is used to override the default location `~/.kube/config`. If it is not set, the default kube config file will be used.